provider/messagebird/messagebird.go

```diff
@@ -26,13 +32,51 @@ func NewMessageBird(config MessageBirdConfig) *MessageBird {
 	}
 	return &MessageBird{
 		client: client,
-		params: messagebird.MessageParams{
+		messageParams: messagebird.MessageParams{
 			Gateway: config.Gateway,
 		},
+		voiceMessageParams: messagebird.VoiceMessageParams{
+			Language: config.Language,
+			Voice: config.Voice,
+			Repeat: config.Repeat,
+		},
 	}
 }
 
 func (mb *MessageBird) Send(message sachet.Message) error {
-	_, err := mb.client.NewMessage(message.From, message.To, message.Text, &mb.params)
-	return err
+	var textRecipients []string
+	var voiceRecipients []string
+
+	for _, recipient := range message.To {
+		targetTypeName := strings.SplitN(recipient, ":", 2)
```

Comment:
The trick of adding a prefix to the recipient string is not great. I think it's not a very strong contract. 
I think we can do better! 

I have a few ideas:

1) if we added another slice to the `sachet.Message` named `VoiceTo`? And those would be the voice messages recipients while the old `To` would still be sms?  
2) Add a "type" field to the `sachet.Message` that can be const "voice" or "sms" and then the providers would act on it
3) Add a different method to the provider interface that is `SendVoice`? 

 Let me know what you think.

Reply:
Hi,
I forgot to mention I basically borrowed the method of using a recipient prefix from the pushbullet provider. I agree this isn't a very great contract, but it's backwards compatible and allows mixing text and voice messages for a single receiver.
However, I think your option 1 would also allow mixing text and voice message recipients with a single receiver. The configuration could be like this:
```yaml
receivers:
  - name: team1
    provider: messagebird
    to:
      - '+917498724'
    voice_to:
      - '+913873892'
    from: '+9128732847'
```
and `sachet.Message` could be extended with a `VoiceTo` field.

We are currently using a variant of your option 2 with a `Type` field added to `sachet.Message`. This doesn't allow mixing text and voice message recipients for a single receiver. Then the configuration could be like this:
```yaml
receivers:
  - name: team1
    provider: messagebird
    type: voice
    to:
      - '+917498724'
    from: '+9128732847'
  - name: team2
    provider: messagebird
    type: text
    to:
      - '+913873892'
    from: '+9128732847'
```
And `type` would default to `text`, so you do not have to add it in the configuration.
I guess how to handle `sachet.Message` is a provider implementation detail in `Send()`

Reply:
I can push our implementation of option 2 to this PR, if you want to, plz let me know.

Reply:
What about leaving `to` for backwards compatibility and the type is always `text` and then add "to_groups" (or something better named)

```yaml
receivers:
  - name: team1
    provider: messagebird
    to_groups:
      - type: voice
        to:
          - "+31621234567"
          - "+31621234568"
      - type: text
        to: 
          - "+31621234567"
          - "+31621234568"
```

Reply:
After checking I think option 2 will be the best.
It won't break anything and could be potentially useful for other provider to have a type flag.

You can go ahead and push it :)